README.md

```diff
@@ -262,6 +262,8 @@ Make sure that you have the following required dependencies installed:
 
 * Clang-format 6.x or higher
 
+* basez (apt-get install basez)
```

Comment:
This apt-get install just for this looks rather arbitrary, in line of other format. Also we don't list jq which we also need, so add that here as well
```suggestion
* jq
* hex (for ubuntu, found in package basez)
```